XRTK-Core/Packages/com.xrtk.core/Extensions/ArrayExtensions.cs

```diff
@@ -37,10 +38,15 @@ public static bool IsValidArray(this Array array)
         /// <param name="array">The array to extend</param>
         /// <param name="newItem">The item to add to the array</param>
         /// <returns></returns>
-        public static T[] AddItem<T>(this T[] array, T newItem, int insertAtIndex = 0)
+        public static T[] AddItem<T>(this T[] array, T newItem)
         {
+            if (array == null)
```

Comment:
need to pull out this change

Reply:
Ahh, I see you modified the original extension, but now need to revert that change

Reply:
As this has been enhanced/resolved in #461 , you will need to revert changes to this file @StephenHodgson .